CONTRIBUTING.md

```diff
@@ -368,6 +369,27 @@ at the beginning of the line:
 I love it! Very clean code and great tests. lgtm.
 ```
 
+## Continuous Integration
+
+The Kata Containers project has a gaiting process to prevent introducing
+regressions. When your PR is submitted, a Continuous Integration (CI) system
+will run different checks on different platforms for your changes. Currently
+we use [Jenkins](http://jenkins.katacontainers.io) and
+[travis](https://travis-ci.org/kata-containers/) for testing your changes.
+
+Some of the checks are:
+
+- Static analysis checks.
+- Unit tests.
+- Functional tests.
+- Integretation tests.
+
+The travis job will be executed right after the PR is opened, while the Jenkins
+jobs will wait to be triggered. A maintainer must add a `/test` comment
+on the PR to let the CI jobs run.
+
```

Comment:
I guess we might need to document '/retest' as well, even though the reality is that it and '/test' are identical.
@chavafg - did you have any further thoughts or details on the possibility of aligning the Jenkins trigger phrases with the Zuul trigger phrases - did we find out if our Zuul trigger phrases can be changed for instance?

Reply:
Found on the Zuul documentation that it can be changed, but not sure we can do it now as the Zuul instance we are running is the one of the Openstack project. Will check further with the openstack-infra team during their weekly meeting later today.

Reply:
@chavafg Is there a command to cancel the builds at an intermediate stage?

Reply:
@ydjainopensource Unfortunately no, at least with the plugin that we currently use. 
@grahamwhaley talking with @cboylan, we wont be able to change Zuul to 'on-demand' in the short term. They have an experimental pipeline where they support this feature, but seems like it is not stable enough and we might get some issues (as per my understanding).